knowage-vue/src/modules/documentExecution/dashboard/widget/WidgetEditor/WidgetEditorSettingsTab/common/interactions/crossNavigation/WidgetOutputParametersList.vue

```diff
@@ -1,5 +1,6 @@
 <template>
     <div class="p-grid">
+        {{ widgetType }}
```

Comment:
Could you please remove the widgetType logging?

Reply:
Done